docs/Permissions/Onchain-Permissioning/Onchain-Permissioning.md

```diff
@@ -0,0 +1,51 @@
+description: Onchain Permissioning
+<!--- END of page meta data -->
+
+# Onchain Permissioning 
+
+Onchain permissioning uses smart contracts to store and maintain the node, account, and admin account whitelists. 
+Using onchain permissioning enables all nodes to read the whitelists from one source.                                        
```

Comment:
I'd replace "read the whitelists from one source." with:
"... read the permissioning rules from a single source of truth, the blockchain."

Reply:
Updated